.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -6,6 +6,7 @@ body:
   attributes:
     value: >
       #### Before submitting a bug, please make sure the issue hasn't been already addressed by searching through [the existing and past issues](https://github.com/pytorch/pytorch/issues?q=is%3Aissue+sort%3Acreated-desc+).
+      #### Please write your bug report in English to ensure it can be understood and addressed by the development team.
```

Comment:
Do you know how this renders? Having two #### lines might be weird...

Reply:
Honestly was going to push and hotfix if it looked weird.